docs/release.md

```diff
@@ -144,6 +145,92 @@ make gotest
 
 in the repository root.
 
+## Add missing upstream components
```

Comment:
@swiatekm-sumo Please look at it :fearful:

Reply:
I updated it. This is valid point about using changelogs of upstream projects :+1: